.github/workflows/auto_approve_prs.yml

```diff
@@ -0,0 +1,88 @@
+name: Auto Approve PRs
+
+on:
+  schedule:
+    # Every 15 minutes
+    - cron: "*/15 * * * *"
+  workflow_dispatch:
+
+concurrency:
+  group: auto-approve-prs
+  cancel-in-progress: false
+
+jobs:
+  find_prs:
+    name: Find PRs to approve
+    runs-on: ubuntu-latest
+    outputs:
```

Comment:
Put outputs below steps. While less conventional, it is easier to read and aligns better with the following job which will consume the output.